client/main.js

```diff
@@ -12,7 +12,7 @@ dpad right
 */
 
 // change the ip to ther current network ip
-ws = new WebSocket('ws://192.168.178.98:7072');
+ws = new WebSocket('ws://127.0.0.1:7072');
```

Comment:
this cant be localhost because you acces the server on lan for example:
the pc is the server and the phone sends data to the server via this websocket